### ISSUE TITLE
Automatically save the text from Whisper recognition as a plain text file

### ISSUE THREAD

**Issue by vivadavid** [SubtitleEdit/subtitleedit#7646]
Hi,

It's obvious that Subtitle Edit is a programme to create and edit subtitles, but the fact is that it's become probably the best tool to manage and use Whisper in its different implementations. For this reason, even if it makes more sense for the text to be automatically saved as a subtitle document, it'd be extremely useful if users could save it instead as a plain text file. I know that you can open an SRT file and export it as a plain text document, but since Subtitle Edit allows for batch processes on Whisper, it would be extremely time-saving to make everything automatic.

I'd add this feature below (or somewhere near) _Use post-processing_, and it could called _Save text as a plain text document_.

Thanks in advance for considering this suggestion.

**Comment by vivadavid** [comment]
Actually it would be even better if the plain text document could be produced **in addition to** the subtitle document. Applying Whisper on audio takes a long of time and resources, and so getting both types of documents in one go, just in case, would be extremely useful.

**Comment by vivadavid** [comment]
I've tried adding this parameter (under _Advanced_) when using Const-me:

-otxt

Unfortunately, it still generates only an SRT file.

I've also tried:

--output-txt

With the same result. 

I don't know if both parameters are interchangeable.

**Comment by vivadavid** [comment]
Hi,

I wanted to gently ask why this suggestion won't be implemented. Is it particularly complex to automatically generate both the SRT and the TXT files? Again, I completely understand this is a subtitling tool, but I find it really useful to have the text file available too (for searching purposes, etc.).

I wish I could, at least, get the TXT file via the advanced settings, but as I said above, the two parameters I've tried won't work (at least, with ConstMe)

Thanks!

**Comment by vivadavid** [comment]
Hi again,

As a workaround, I was trying two alternative ways, but I bumped into problems, so I wanted to know if I'm doing something wrong:

1. As far as I can see, if I use the _Export_ tool to save my SRT files as plain text, I have to do it one file at a time.
2. If I use the _Batch convert_ tool, I can't find a way to export my subtitles as plain text.

Would there be a way to combine both tools, so I can generate plain text files as a batch process?

Thanks!

**Comment by niksedk** [comment]
You can batch convert to plain text:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/a9b350a7-13de-4843-9434-8a6efff63f04)

**Comment by vivadavid** [comment]
Thank you for your reply, @niksedk ,

I didn't understand what was going on, as I had gone through all the formats, but after a while, I've realized that I stoppped scrolling after the first 10 or 20 unknowns formats, thinking that the rest were also unknown. Would it make sense to place all the unknown formats at the end of the list?

I've done a batch convert to the plain format and I'm very happy with the result and with this solution in general: thank you!
